### PR TITLE
Welcome Tour: Fix overlay controls overlapping 

### LINKED ISSUE #59244: Welcome Tour: Explore new design for Header controls
### What

[Introducing new gifs ](https://github.com/Automattic/wp-calypso/pull/56347) for the mobile Welcome Tour, we encountered some problems with the tour overlay controls overlapping with the gifs.

We shipped a [quick solution](https://github.com/Automattic/wp-calypso/pull/59052), but that's not optimal.

We talked about this in the pod slack thread: p1639497273123400-slack-C029WSAQMT9

### How

I'll try to cover in bullet points the conversation we had on Slack and what we should do:

- Header controls overlap with imgs/gifs. Possible solutions:
-- Background image with negative margin
-- ...
- Header controls may be too big, grabbing a bit too much attention.
- Hover effect on desktop (non-touch enabled devices): 
-- Might be an accessibility issue
-- Not optimal CSS solution used for that ([56164](https://github.com/Automattic/wp-calypso/pull/56164))
-- Always showing the controls seems like a more modern solution

### PR DESCRIPTION
**Pull request by escapemanuele**
#### Changes proposed in this Pull Request

TBD

#### Testing instructions

TBD

Related to #59244
Fixes #59244

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js [hunk 1]
```diff
@@ -122,7 +124,9 @@ function WelcomeTour() {
 					[]
 				),
 			],
-			className: 'wpcom-editor-welcome-tour',
+			className: classNames( 'wpcom-editor-welcome-tour', {
+				'is-desktop': ! isMobile(),
```
Comment (chriskmnds) [inline apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js:128]: I think we can add both `is-mobile` & `is-desktop` classes to the tour kit directly, as these are fairly generic concepts that any implementation could benefit from. (could be attached to `tour-kit-frame` - but please check if another parent is better)

p.s. if we can do the above in a PR (this or another) without the welcome-tour modifications, then even better. So one to introduce the viewport classes, and another to follow up on the `absolute` positioning part. In fact, this PR (or whichever "welcome-tour" PR we use) could really instead just remove the `absolute` positioning and focus on the proper fix instead.

**Review by chriskmnds — COMMENTED** [review]
Thanks for the follow-up @escapemanuele. I left a comment below. If we take that approach (of 2 PRs), then we can update the labels and PR title accordingly (add "Tour Kit" where relevant, etc.) 🙇

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js (hunk 1 above)
Comment (escapemanuele) [reply apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js:128]: Thank you @chriskmnds! 😄 
That's an interesting idea, I'll create another PR for introducing the viewport classes, this one instead will be used to fix the actual problem 🚀

**Comment by escapemanuele** [comment]
Closing as this problem has already been solved with [59450](https://github.com/Automattic/wp-calypso/pull/59450) and [59462](https://github.com/Automattic/wp-calypso/pull/59462)

**Pull request closed without merging**